### PR TITLE
Changes to template of hotelsCombined tag

### PR DESCRIPTION
**Pull request by srkurup**

### CODE REVIEW

**Comment by srkurup** [comment]
rate

File: src/profile/main/tag/hotelsCombined.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ try{
 
       if(typeof b._corder!="undefined"){
 	//c.push("BookingValue="+b._csubtotal);
-	c.push("BookingValue="+b.checkout.hotel.totalRoomRate);
+	c.push("BookingValue="+b.entity.checkout.hotel.rooms[0].totalPrice.amount);
 	//c.push("Currency="+((b._ccurrency)?b._ccurrency:"USD"));
```
Comment (j-ibrahim) [inline src/profile/main/tag/hotelsCombined.js]: @srkurup , please check the legacy implementation for hotelsCombined to confirm whether total booking Cost or Room Rate need to be passed

**Comment by srkurup** [comment]
carttotal

**Comment by srkurup** [comment]
carttotal

**Review by j-ibrahim — APPROVED** [review]

**Pull request merged**